### PR TITLE
solution

### PR DESCRIPTION
**Pull request by yelnikov-andrii**

### CODE REVIEW

File: src/makeCalculator.js [hunk 1]
```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    operate(func, value) {
```
Comment (valerii-derkach) [inline src/makeCalculator.js:42]: How the calculator will work:
 * - Each `operate` call takes a callback and a number and **sets the
 *   appropriate value to the `result` property.**
in your case the result property is changing in methods: `add`, `subtract`, `multiply`, `divide`, but not in 'operate' method

**Review by valerii-derkach — CHANGES_REQUESTED** [review]

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]
Fix @valerii-derkach comment

**Comment by yelnikov-andrii** [comment]
> Fix @valerii-derkach comment

I don't understand this problem, all tests are working

**Review by Anna-Tkachenko — APPROVED** [review]
Good job!

**Pull request closed without merging**